buildscripts/cstor-volume-mgmt/entrypoint.sh

```diff
@@ -2,6 +2,5 @@
 
 set -ex
 
-exec /usr/local/bin/cstor-volume-mgmt start
-exec service ssh start
-exec service rsyslog start
+/usr/local/bin/cstor-volume-mgmt start &
+exec /usr/local/bin/cstor-volume-grpc server start
```

Comment:
nit : exec should be at start of the command line ?

Reply:
if we put exec for the first line, the shell process will be replaced by the process that we invoked. it will not execute the following commands. that is why i used exec in the last line only.